### ISSUE TITLE
3D printing file

### ISSUE THREAD

**Issue by hamadfayyad** [simonemarin/esphome_smartlock#1]
Could you please provide me with a 3D printing file

Thanks!

**Comment by simonemarin** [comment]
Yes I can provide .stl files but do remember that you can easily export them from Blender, which is free and that they will probably not fit your door.

**Comment by simonemarin** [comment]
There you go
